templates/CRM/Profile/Page/MultipleRecordFieldsListing.tpl

```diff
@@ -33,7 +33,7 @@
               {literal}
               <script type="text/javascript">
                 (function($) {
-                  var ZeroRecordText = {/literal}'{ts 1=$customGroupTitle|escape}No records of type \'%1\' found.{/ts}'{literal};
+                  var ZeroRecordText = {/literal}"{ts escape='js' 1=$customGroupTitle}No records of type '%1' found.{/ts}"{literal};
```

Comment:
I think you might as well make it `{ts escape='js' 1=$customGroupTitle|smarty:nodefaults}` - since it's a good guess this would break under strict smarty too

Reply:
@eileenmcnaughton Good idea. I've `r-run` with that and force-pushed.

Also, the branch should be amenable to 5.50, so I'll switch it.